docs/guides/integration-mobile-client-side.md

```diff
@@ -67,7 +67,7 @@ Follow the applicable instructions, for Android or iOS:
 
 1. Check out the main branch of the [SDK for Android source code repository on GitHub](https://github.com/IABTechLab/uid2-android-sdk/tree/main).
 1. In Android Studio (check the version required in the [Minimum Requirements](../sdks/sdk-ref-android.md#minimum-requirements) section in the SDK for Android Reference Guide), open the directory that you checked out.
-1. set `uid2_environment_euid` to `false` in [AndroidManifest.xml](https://github.com/IABTechLab/uid2-android-sdk/blob/main/dev-app/src/main/AndroidManifest.xml).
+1. In [AndroidManifest.xml](https://github.com/IABTechLab/uid2-android-sdk/blob/main/dev-app/src/main/AndroidManifest.xml), set `uid2_environment_euid` to `false`. [**GWH_CM01 is this correct, uid2 and euid?**]
```

Comment:
Do we want to keep this? [**GWH_CM01 is this correct, uid2 and euid?**]

Reply:
@caroline-ttd Aargh. I owe you on that one. Thx.